module/account/src/Application/Security/Voter/UserRoleVoter.php

```diff
@@ -45,6 +48,10 @@ public function supports($attribute, $subject): bool
      */
     public function voteOnAttribute($attribute, $subject, TokenInterface $token): bool
     {
+        @trigger_error(
+            'Used UserRoleVoter is deprecated and will throw Fatal error in 2.0. You should use ErgonodeRoleVoter',
```

Comment:
I'd stick to a more descriptive explanation like that one should use roles starting with `ERGONODE_ROLE_*`